### DISCUSSION TITLE
How to deal with automatic capitalization of url

### DISCUSSION THREAD

**Discussion by daniyalmujtaba** [remix-run/react-router#8633] [Q&A] [answered]
I am new to web dev and routing structures and I am using react router v6 in my project. I am facing one issue where I am passing a url param is lower case and its automatically capitalized. I am even using caseSensitive in <Route/> but its being capitalized. is there any way to deal with this?

**Comment by timdorr** [comment, marked as answer]
That isn't a feature of React Router, so something somewhere else is transforming those params to upper case.

**Reply by daniyalmujtaba** [reply]
Yeah it was browser cache that was messing it up. I used it with capitalization for the first time and it kept switching to it. once I cleared the browser storage of my project and it worked fine.

**Reply by pablojsx** [reply]
happening to me now, with nextjs....
